components/pages/Index.tsx

```diff
@@ -0,0 +1,19 @@
+import RulesBtn from "../button/RulesBtn";
+import StartGameBtn from "../button/StartGameBtn";
+import Logo from "../logo/Logo";
+import "./Index.module.css";
+import styles from "./Index.module.css";
+
+function IndexPage() {
+  return (
+    <div className={styles.container}>
+      <Logo size="big" />
+      <div className={styles.menuItems}>
+        <StartGameBtn />
+        <RulesBtn />
+      </div>
+    </div>
+  );
+}
+
+export default IndexPage;
```

Comment:
I wouldn't create this in the `components` folder. This is a page and should be created directly in the `pages` folder